modules/upgrade/pages/k-upgrade-kubernetes.adoc

```diff
@@ -81,7 +81,7 @@ Before you upgrade a worker node, make sure that it is cordoned to prevent new P
 * https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster[Upgrade on GKE (Google Kubernetes Engine)^]
 * https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html[Upgrade on EKS (Elastic Kubernetes Service)^]
 * https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli[Upgrade on AKS (Azure Kubernetes Service)^]
-* https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/[Upgrade on Self-Hosted^]
+* https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/[Upgrade on self-managed^]
```

Comment:
@micheleRP in this case should both be capital? 
```suggestion
* https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/[Upgrade on Self-Managed^]
```

Reply:
yes, please!